aptos-move/aptos-vm/src/data_cache.rs

```diff
@@ -98,6 +99,27 @@ impl<'block, S: StateView> StateView for StateViewCache<'block, S> {
         }
     }
 
+    fn get_state_value(&self, state_key: &StateKey) -> anyhow::Result<Option<Vec<u8>>> {
```

Comment:
I slightly prefer to call this just `get()`, just saying :P

Reply:
I would like that too but at this point that would be ambigous. Let's rename this to `get()` once we completely move to the fine-grained storage :)